src/style.css

```diff
@@ -1 +1,76 @@
-/* Styles go here */
+html {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 14px;
+  text-transform: uppercase;
+
+  --somecolor: #00acdc;
```

Comment:
please give better naming to your CSS variable to understand what the color is used for. and poot it into `:root` selector